onnx/defs/tensor/utils.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) ONNX Project Contributors.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "onnx/defs/schema.h"
+#include "onnx/defs/tensor_proto_util.h"
+
+#include <cmath>
+
+namespace ONNX_NAMESPACE {
+  void resizeShapeInference(InferenceContext& ctx);
+  void upsampleShapeInferenceV7(InferenceContext& ctx);
```

Comment:
is this one "upsampleShapeInferenceV7" also needed to be generalized and put in this common file? I'm seeing it's only used once. if it's not needed, let's not put it here,